packages/ui-components/src/layouts/ResultsLayout/index.tsx

```diff
@@ -22,6 +23,7 @@ export const ResultsLayout = compose(
     view: {
       pagination: true,
       infinite: false,
+      stickyFacets: true,
```

Comment:
as you asked in slack @Bitaru

Reply:
ah, it should be disabled by default as I remember

Reply:
Yeah, and also we need to be sure iframe is disabled